README.md

```diff
@@ -71,30 +71,13 @@ Sample launch configuration files are in the [examples directory](https://github
 In the event that the installation is a mix of Debian and from source, command
 line tools from `gz-tools` may not work correctly.
 
-A workaround for a single package is to define the environment variable
-`GZ_CONFIG_PATH` to point to the location of the Gazebo library installation,
-where the YAML file for the package is found, such as
+A workaround is to define the environment variable
+`GZ_CONFIG_PATH` to point to the different locations of the Gazebo libraries installations,
+where the YAML files for the packages are found, such as
 ```
-export GZ_CONFIG_PATH=/usr/local/share/gz
+export GZ_CONFIG_PATH=/usr/local/share/gz:$HOME/.gz/tools/configs
```

Comment:
The `$HOME/.gz/tools/configs` directory is created as part of the workaround in the previous version. If we're removing that, it won't make much sense here. Instead I'd suggest this
```suggestion
export GZ_CONFIG_PATH=/usr/local/share/gz:$HOME/ws/install/share/gz
```

and say "where `$HOME/ws` is an example colcon workspace used to build Gazebo"